cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js

```diff
@@ -343,15 +343,15 @@ When('you change the sharing settings', () =>
                 cy.contains('Share view')
                     .click();
 
-                cy.get('[placeholder="Enter names"]')
+                cy.get('[placeholder="Search"]')
```

Comment:
Hey, 

There are 2 comments related to sharing dialog on lines [313](https://github.com/dhis2/capture-app/blob/master/cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js#L312) and line [399](https://github.com/dhis2/capture-app/blob/master/cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js#L399). It looks like, the request had to be made manually from within Cypress to be able to test the dialog. 
Now, after the refactoring done in this PR, I believe that the dialog can be properly tested by spying on the real requests done by the `SharingDialog` and `useDataQuery`. WHYT?

Thank you!